### PR TITLE
docs: Paging with OFFSET and LIMIT

### PR DESCRIPTION
**Pull request by johanandren**
Covers https://github.com/lightbend/akkaserverless-framework/pull/1043 and https://github.com/lightbend/akkaserverless-framework/pull/1049 so no need to merge until those are released and deployed.

I think the query language overview should move in the main docs and not in the SDK since it will be the same across SDKs.

### CODE REVIEW

File: docs/src/modules/java/pages/views.adoc [hunk 1]
```diff
@@ -486,7 +486,9 @@ Or come from fields in the request message:
 SELECT * FROM customers OFFSET :start_from LIMIT :max_customers
 ----
 
-Note: `OFFSET` and `LIMIT` cannot be used in queries containing `OR`.
+Note: Using numeric offsets can lead to missing or duplicated entries in the result if entries are added to or removed from the view between requests for the pages. The token based paging handles this better.
```
Comment (raboof) [inline docs/src/modules/java/pages/views.adoc:489]: Perhaps we should just describe in what cases entries are duplicated/missing without pointing to token-based paging? token-based paging is not *much* better, after all...

Comment (johanandren) [reply docs/src/modules/java/pages/views.adoc:489]: Lets' remove the token based reference, but not sure I agree we should spend doc-space detailing exact scenarios.

**Comment by johanandren** [comment]
Can't be merged until we transition to the new database, or needs the token based paging separated since only index based paging works with spanner.

**Comment by johanandren** [comment]
Now only contains the index based paging

**Review by ennru — APPROVED** [review]
LGTM.

**Pull request merged**